qiskit_nature/second_q/properties/bases/electronic_basis_transform.py

```diff
@@ -65,7 +62,7 @@ def is_alpha_equal_beta(self) -> bool:
         return np.allclose(self.coeff_alpha, self.coeff_beta)
 
     def __str__(self) -> str:
-        string = [super().__str__() + ":"]
+        string = ["ElectronicBasisTransform:"]
```

Comment:
Should we prefer to use self.name here (since thats set to the class name) rather than hard code this?

Reply:
This will get refactored further to a degree where this will no longer be a property. Instead, this will be implemented as a `Transformer` at which point this method might go away entirely